netmiko/huawei/huawei_smartax.py

```diff
@@ -0,0 +1,80 @@
+import time
+import re
+from netmiko.cisco_base_connection import CiscoBaseConnection
+from netmiko import log
+
+
+class HuaweiSmartAXSSH(CiscoBaseConnection):
+    def session_preparation(self):
+        """Prepare the session after the connection has been established."""
+        self.ansi_escape_codes = True
+        self._test_channel_read()
+        self.set_base_prompt()
+        self.disable_smart_interaction()
```

Comment:
Convert this to a private method i.e. add a leading underscore `self._disable_smart_interaction()`